README.md

```diff
@@ -1,8 +1,70 @@
 # pulp-2to3-migration-test-fixtures
 
-This repository contains a mongodump archives, /var/lib/pulp/content archives, and /var/lib/published archives.
+This repository contains Pulp 2 snapshots in the form of mongodump archives, /var/lib/pulp/content
+ archives, and /var/lib/pulp/published archives to used in tests for the pulp-2to3-migration plugin.
+ 
 
-The following commands were used to generated the current snapshots. 
+### Structure
+
+There is a separate directory for each Pulp 2 snapshot.
+Each directory has to have:
+ - a mongodump
+ - a /var/lib/pulp/content snapshot
+ - a /var/lib/pulp/published snapshot
+ 
```

Comment:
- a config to generate the Pulp 2 snapshot

Reply:
it's not among mandatory ones, so it's mentioned in the sentence below.